src/inc/msquic.h

```diff
@@ -663,7 +664,8 @@ typedef struct QUIC_SETTINGS {
             uint64_t GreaseQuicBitEnabled                   : 1;
             uint64_t EcnEnabled                             : 1;
             uint64_t HyStartEnabled                         : 1;
-            uint64_t RESERVED                               : 29;
+            uint64_t EncryptionOffloadEnabled               : 1;
```

Comment:
Let's wrap the `QUIC_SETTINGS` changes in `#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES` checks for now. You can leave the stats one without.